services/workflows-service/src/alert/types.ts

```diff
@@ -5,3 +5,33 @@ export type TAuthenticationConfiguration = {
   isValid: boolean;
   webhookSharedSecret: string;
 };
+
+export const BulkStatus = {
+  success: 'success',
+  partial: 'partial',
+  failed: 'failed',
+} as const;
+
+export type TBulkStatus = (typeof BulkStatus)[keyof typeof BulkStatus];
+
+export type TAlertUpdateResponse = {
+  alertId: string;
+  status: string;
+}[];
+
+type SuccessResponse = {
+  alertId: string;
+  status: 'success';
+};
+
+type FailedResponse = {
+  alertId: string;
+  status: 'failed';
+  error: {
+    message: string;
+  };
```

Comment:
Don't think errors should be handled differently in different places.
```suggestion
  errors: {
    path?: string;
    message: string;
  }[];
```

Reply:
its not a validation error, this kind of error handling by class-validator before reaching the Controller api

Reply:
I understand where you're coming from but as a client it would be unpredictable and require more inside knowledge (even if from Swagger) if sometimes its `error` sometimes `errors` with different data structures.

Reply:
Please resolve and we will discuss about it later with alon